script/Ciel/Day 14/14_01B_01_CIEL14B_1D.txt

```diff
@@ -951,12 +951,12 @@ This too had been part of Ciel's predictions.
 [sha:0ea02769c30b0ad12e349dcfbc75e553e9b213ba]{
 -- Page 200, Offset 38914.
 -- まず頭を潰して意識を刈り取る。
-First, she would take out her head and render her unconscious.
+First, Ciel would take out Arcueid's head and render her unconscious.
 }
 [sha:e0f9af5d600cf3a612a3ac4b196b71422fe2573f]{
 -- Page 200, Offset 38915.
 -- その後、彼女が思考能力を取り戻す前に本体を破壊する。
-Then, before Arcueid regains the ability to think clearly, she would obliterate her body.
+Then, before Arcueid regains the ability to think clearly, Ciel would obliterate her body.
```

Comment:
I think keep her -> Arcueid in the first sentence, but leave the `she`s. Should be clear from context of prev page that Ciel is doing the actions.